radio/src/opentx.h

```diff
@@ -1385,6 +1387,10 @@ union ReusableBuffer
 #endif
   } modelsel;
 
+  struct {
+    char msg[64];
+  } msgbuf;
```

Comment:
it should be modelsetup instead of msgbuf, to assert we don't have 2 temp buffers for the same screen